doc/build/orm/loading_columns.rst

```diff
@@ -253,7 +269,8 @@ Deferred "raiseload" can be configured at the mapper level via
         summary = deferred(Column(String(2000)), raiseload=True)
         excerpt = deferred(Column(Text), raiseload=True)
 
-    book_w_excerpt = session.query(Book).options(undefer(Book.excerpt)).first()
+    book_w_excerpt = session.scalars(select(Book).options(undefer(Book.excerpt))).first()
```

Comment:
**mike bayer** (zzzeek) wrote:

.limit(1)

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3729

Reply:
Changed